### PR TITLE
fix(form): fix no effect when label-width prop is auto

### LINKED ISSUE #2479: [Bug Report] Form does not take effect when label-width prop is auto
Element Plus version
1.0.2-beta.54

OS/Browsers version
Window10/Chrome 91.0.4472.124

Vue version
3.1.4

Reproduction Link
https://codepen.io/qige/pen/YzVWvyN

Steps to reproduce
Set label-width to auto

What is Expected?
Label-width calculation is right

What is actually happening?
Label-width calculation is no effect

### PR DESCRIPTION
**Pull request by qige2016**
fix #2479 

Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by kooriookami** [comment]
@qige2016 Hi, set `label-position="left"` does not work, can you fix it and add a test?
image

File: packages/form/src/form-item.vue [hunk 1]
```diff
@@ -130,24 +130,23 @@ export default defineComponent({
 
     const labelFor = computed(() => props.for || props.prop)
     const labelStyle = computed(() => {
-      if (elForm.labelPosition === 'top') return {}
+      const ret: Partial<CSSStyleDeclaration> = {}
```
Comment (JeremyWuuuuu) [inline packages/form/src/form-item.vue:133]: Actually we can use `CSSProperties` from `Vue` bundle

```ts
import type { CSSProperties } from 'vue'
```

**Comment by qige2016** [comment]
@kooriookami @JeremyWuuuuu done~

File: packages/form/src/form-item.vue [hunk 2]
```diff
@@ -54,6 +54,7 @@ import { elFormEvents, elFormItemKey, elFormKey } from './token'
 import type { PropType } from 'vue'
 import type { ElFormContext, ValidateFieldCallback } from './token'
 import type { FormItemRule } from './form.type'
+import type { CSSProperties } from 'vue'
```
Comment (JeremyWuuuuu) [inline packages/form/src/form-item.vue:57]: Minor: you can add this member at line#54

Comment (JeremyWuuuuu) [reply packages/form/src/form-item.vue:57]: @qige2016 Please address this one so that we can get this merged.

Comment (qige2016) [reply packages/form/src/form-item.vue]: @JeremyWuuuuu Thanks for reviewing.

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
LGTM.

**Review by kooriookami — APPROVED** [review]

**Pull request merged**